src/firejail/usage.c

```diff
@@ -201,6 +201,7 @@ static char *usage_str =
 	"    --private-home=file,directory - build a new user home in a temporary\n"
 	"\tfilesystem, and copy the files and directories in the list in\n"
 	"\tthe new home.\n"
+	"    --keep-shell-rc - do not copy shell configuration from /etc/skel\n"
```

Comment:
The options are sorted alphabetically in this file; move it below `--keep-dev-shm`.

Reply:
Makes sense, I moved it below the `--keep-fd`, to keep the alphanumerical ordering.